draft-ietf-tls-ctls.md

```diff
@@ -305,6 +305,7 @@ Value: a single `CTLSExtensionTemplate` struct:
 struct {
   Extension predefined_extensions<0..2^16-1>;
   ExtensionType expected_extensions<0..2^16-1>;
+  ExtensionType known_self_delimiting_extensions<0..2^16-1>;
```

Comment:
Let's try to make this shorter.

```suggestion
  ExtensionType omit_length<0..2^16-1>;
```

Other possibilities: `static`, `self_delimiting_extensions`.